odl/test/tomo/operators/ray_trafo_test.py

```diff
@@ -191,9 +191,9 @@ def test_projector(projector, in_place):
 def test_adjoint(projector):
     """Test discrete Ray transform backward projection."""
 
-    # TODO: this needs to be improved
-    # Accept 10% errors
-    places = 1
+    if isinstance(projector.geometry, odl.tomo.CircularConeFlatGeometry):
```

Comment:
See comment further down, `CircularConeFlatGeometry` is a special case of helical and should never be treated as a separate class in ODL. If you want ot handle it, check if it is a helical with pitch 0.